spec/controllers/games_controller_spec.rb

```diff
@@ -0,0 +1,120 @@
+require 'rails_helper'
+
+RSpec.describe GamesController, type: :controller do
+
+  describe "games#new action" do 
+    it "should require users to be logged in" do 
+      get :new
+      expect(response).to redirect_to new_user_session_path
+    end
+    
+    it "should return success response on the get request" do
+      user = create(:user)
+      sign_in user
+      get :new
+      expect(response).to have_http_status(:success)
+    end
+  end
+
+  describe "games#show action" do 
+    it "should return success response on the get request" do
+      game = create(:game)
+      get :show, id: game.id
+      expect(response).to have_http_status(:success)
+    end
+
+    it "should return a Game Not Found error if the game is not found" do 
+      get :show, id: '99999999'
+      expect(response).to have_http_status(:not_found)
+    end
+  end
+
+  describe "games#update action" do 
+    before :each do 
+      @game = FactoryGirl.create(:game, name: "Test Name", state: "open")
+    end
```

Comment:
When possible, it's generally considered better to use `let` in these scenarios rather than `before` and an instance variable: http://stackoverflow.com/questions/5359558/when-to-use-rspec-let. See here for info on `let`: http://www.relishapp.com/rspec/rspec-core/v/3-4/docs/helper-methods/let-and-let